plugin/trino-kafka-event-listener/src/main/java/io/trino/plugin/eventlistener/kafka/KafkaEventListenerConfig.java

```diff
@@ -47,7 +47,7 @@ public class KafkaEventListenerConfig
     private Set<String> excludedFields = Collections.emptySet();
     private Map<String, String> kafkaClientOverrides = Collections.emptyMap();
     private Duration requestTimeout = new Duration(10, SECONDS);
-    private boolean terminateOnInitializationFailure = true;
```

Comment:
We probably could just keep this commit, but I still don't understand the rationale? Does it fail in tests or something else?

Reply:
Not a 100% sure this is a good change, but I can be convinced. I think that if a user explicitly configures a kafka-event-listener, they would want to know if the brokers are unreachable instead of the plugin silently "failing"